R/dfm_weight.R

```diff
@@ -112,6 +120,8 @@ dfm_weight.dfm <- function(
         } else if (scheme == "tfidf") {
             .Deprecated(msg = 'scheme = "tfidf" is deprecated; use dfm_tfidf(x) instead')
             return(dfm_tfidf(x, base = base))
+        } else if (scheme == "logsmooth") {
+            return(dfm_smooth(x, smoothing) %>% log(base = base) %>% as.dfm())
```

Comment:
Let's not induce unnecessary dependency on `%>%`. **quanteda** should not fail because of a bug convenience packages.

Reply:
We re-export (from **magrittr**) it already, and it's such an integral part of many packages that it's unlikely to change.

But I'm indifferent. Is this better?
```r
return(as.dfm(log(dfm_smooth(x, smoothing), base = base)))
```

Reply:
Exporting magrittr is fine (I use it myself outside of functions), because we don't use `%>%` when it is broken.